Src/Support/Google.Apis.Auth/OAuth2/IGoogleCredential.cs

```diff
@@ -42,11 +42,15 @@ internal interface IGoogleCredential : ICredential, ITokenAccessWithHeaders
         IGoogleCredential WithQuotaProject(string quotaProject);
 
         /// <summary>
-        /// Returns true only if this credential type has no scopes by default and requires
-        /// a call to <see cref="MaybeWithScopes(IEnumerable{string})"/> before use.
-        /// See <see cref="GoogleCredential.IsCreateScopedRequired"/> for more information.
+        /// Returns true if this credential has scopes that have been explicitly set
```

Comment:
Hmm... I'm not sure about the "or if it is a type that is scoped by default". I'd expect to be able to find the scopes of such credentials (e.g. ComputeCredentials) and I don't think I can. (And if I'm right about there being a possibility of creating a scoped compute credential, we'd want to be able to distinguish those.)

I may well be wrong on this one though - happy to talk more tomorrow.

Reply:
Ah yes, I'm pretty sure you can create a scoped Compute credential, we just don't support it, and the comments refer to our implementation. But I see how it may be confusing. I'll try to rephrase to make it all clearer.